lib/bolt/cli.rb

```diff
@@ -163,8 +162,16 @@ def parse
 
       warn_inventory_overrides_cli(options)
       options
+
+      # This needs to happen here to avoid a regression where error messages from
+      # loading inventory aren't printed
     rescue Bolt::Error => e
-      outputter.fatal_error(e)
+      if $stdout.isatty
+        # Print the error message in red, mimicking outputter.fatal_error
+        $stdout.puts("\033[31m#{e.message}\033[0m")
+      else
+        $stdout.puts(e.message)
+      end
```

Comment:
We shouldn't replace using the outputter here, as errors raised after the config is loaded will no longer provide the same information they did before this change. For example, when using JSON output, having a target/group name conflict will now only show this:

```
Group linux conflicts with target of the same name for group at ["linux", "all"]
```

When it used to show this:
```
{"kind":"bolt.inventory/validation-error","msg":"Group linux conflicts with target of the same name for group at [\"linux\", \"all\"]","details":{"path":["linux","all"]}}
```

We should instead have the outputter still handle all errors raised after the config is loaded and only print directly to stdout when it is not yet loaded.

Reply:
Using the human outputter that doesn't work though - if there's an error in inventory the message gets dropped.

Reply:
I get it we need two rescues. fuck me.